docs/source/workflow_inputs.rst

```diff
@@ -682,12 +686,17 @@ Wall/Floor Furnace
 
 If a wall furnace or floor furnace is specified, additional information is entered in ``HeatingSystem``.
 
-  ===============================================  ======  =====  ===========  ========  =======  ===================
-  Element                                          Type    Units  Constraints  Required  Default  Notes
-  ===============================================  ======  =====  ===========  ========  =======  ===================
-  ``AnnualHeatingEfficiency[Units="AFUE"]/Value``  double  frac   0 - 1        Yes                Rated efficiency
-  ``extension/FanPowerWatts``                      double  W      >= 0         No        0        Fan power
-  ===============================================  ======  =====  ===========  ========  =======  ===================
+  ===============================================  =======  =====  ===========  ========  ========  =================================
+  Element                                          Type     Units  Constraints  Required  Default   Notes
+  ===============================================  =======  =====  ===========  ========  ========  =================================
+  ``YearInstalled``                                integer                      See [#]_            Year installed
+  ``AnnualHeatingEfficiency[Units="AFUE"]/Value``  double   frac   0 - 1        See [#]_  See [#]_  Rated efficiency
```

Comment:
I suggest combining these into a single row with an "or" and marking it as required. For example, look at what we did for PV (SystemLossesFraction or YearModulesManufactured). And then you can combine the first too table endnotes into one.

(Make the same change for other HVAC system types.)